etc/make-lint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I suppose this intentionally doesn't have `set -e` at the start so that it continues even if something fails?

Reply:
Yeah that's right. Then we test for non-zero exit status as we go, but only exit out once all of the linting steps are done.